lib/stm32/f3/adc.c

```diff
@@ -596,6 +596,35 @@ void adc_set_clk_prescale(uint32_t adc, uint32_t prescale)
 	ADC_CCR(adc) = reg32;
 }
 
+/*---------------------------------------------------------------------------*/
+/** @brief Set Delay between 2 sampling phases in Dual Mode
+ *
+ * This delay is used in dual interleaved mode.
+ *
+ * Actual delay depends of the resolution ADC.
+
+ * The delay is capped to a maximum value which depends of the ADC resolution:
+ * - 12 bit: ADC_CCR_DELAY_12ADCCLK
+ * - 10 bit: ADC_CCR_DELAY_10ADCCLK
+ * -  8 bit: ADC_CCR_DELAY_8ADCCLK
+ * -  6 bit: ADC_CCR_DELAY_6ADCCLK
```

Comment:
this could all just be "12 clocks" if you drop all the defines, and you can just talk about the range.